playground/main.ts

```diff
@@ -3,7 +3,8 @@
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const signinWidgetOptions = require('../.widgetrc.js'); // commonJS module
 
-// import OktaSignIn from '../src/widget/OktaSignIn';
+import { OktaA11yPlugin } from '../src/plugins/OktaA11yPlugin';
```

Comment:
Can playground access this plugin via global OktaPlugins instead of the src dir? maybe add another script tag in index.html

Reply:
Good idea, I'll give it a try